Makefile.am

```diff
@@ -594,7 +594,7 @@ netdata_SOURCES = $(NETDATA_FILES)
 
 if ENABLE_ACLK
 netdata_LDADD = \
-    externaldeps/mosquitto/lib/libmosquitto.a \
+    externaldeps/mosquitto/libmosquitto.a \
```

Comment:
The current version of #8025 actually does use `externaldeps/mosquitto/lib` for these.

Reply:
are you sure? i just run it and it doesn't create lib folder.

Reply:
Ah, no, you're right. I forgot what I had changed when updating it to use `externaldeps`.